addon/-private/object.ts

```diff
@@ -1,31 +1,77 @@
 import { assert } from '@ember/debug';
-
+import { DEBUG } from '@glimmer/env';
 import { USE_PROXY } from './config';
 import { consume, dirty, SUPPORTS_PROXY } from './util';
 
+const setterPropertyPrefix = '__PROPERTY__';
+
+function defineHiddenProperty(obj: object, propertyName: string, value: any) {
+  Object.defineProperty(obj, propertyName, {
+    enumerable: false,
+    configurable: true,
+    writable: true,
+    value
+  })
+}
+
+function patchObjectDescriptor(obj: object, keyName: string, desc: PropertyDescriptor) {
+  if (desc.configurable === false) {
+    return;
+  }
+  if (typeof desc.get === 'function' || typeof desc.set === 'function') {
+    return;
```

Comment:
```suggestion
    Object.defineProperty(obj, keyName, desc);
```

This way, the new `TrackedObject` will have the getter/setter, but it won't track changes to it, which matches the semantics of tracked accessors elsewhere. Sorry if this doesn't make sense, it will in the long run as we start to add more tooling for tracked properties 😄